domain-proxy/Dockerfile.all-in-one

```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine AS builder
```

Comment:
Can this not be from a RH golang?

Reply:
The only option is [rhel8/go-toolset](https://catalog.redhat.com/software/containers/rhel8/go-toolset/5b9c810add19c70b45cbd666), which requires either use of registry tokens or RH login to pull.

Reply:
Which apparently works just fine...